.github/workflows/FirebaseDeploy.yml

```
@@ -0,0 +1,57 @@
+      - name: Install Dependencies
+        working-directory: Splito/functions
+        run: npm ci
+
+      - name: Install Firebase CLI
+        run: npm install -g firebase-tools
+
+      - name: henticate Firebase using service account
+        env:
+          FIREBASE_PROJECT_ID: ${{ secrets.FIREBASE_PROJECT_ID }}
+          FIREBASE_SERVICE_JSON: ${{ secrets.FIREBASE_SERVICE_JSON }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)